docs/theme/index.hbs

```diff
@@ -0,0 +1,403 @@
+<!DOCTYPE HTML>
```

Comment:
This will get out of date if the default mdboox index changes. Is there a way to insert the new button into the existing theme? If not, then I guess this is the way to go.